### DISCUSSION TITLE
About admin panel

### DISCUSSION THREAD

**Discussion by mezut35** [blockscout/blockscout#9395] [Support]
I've set ADMIN_PANEL_ENABLED: "true" but I can't figure out how to direct it to an endpoint. I tried accessing /admin but was unsuccessful. I couldn't find anything very clear in the document at https://github.com/blockscout/blockscout/pull/5208. Could you please explain to me how to proceed on this matter? Thank you in advance for your attention.

**Comment by jacko06v** [comment]
idem
